distribution/test/itests/test-itests-common/src/main/java/ddf/common/test/csw/mock/FederatedCswMockServer.java

```diff
@@ -48,9 +47,11 @@
 
     private StubServer cswStubServer;
 
-    private String defaultCapabilityResponseResourceName = "csw-get-capabilities-response.xml";
+    private String defaultCapabilityResponse = getDefaultResponseResource(
```

Comment:
Yes, this class was coupled to resource paths because of this line https://github.com/codice/ddf/pull/1256/files/d4427a31a716357d5bb59c99a431b7b4326183a5#diff-ca4d0668a08c0921b2cc334b43287a88L184. This makes it difficult to use outside of this module, so I abstracted it to use content instead.